packages/lesswrong/server/callbacks/commentCallbacks.ts

```diff
@@ -243,35 +244,21 @@ async function sendModerationPM({ firstMessageContents, lwAccount, comment, noEm
     validate: false
   });
 
-  const firstMessageData = {
+  const messageData = {
     userId: lwAccount._id,
     contents: {
       originalContents: {
         type: "html",
-        data: firstMessageContents
+        data: messageContents
       }
     },
     conversationId: conversation.data._id,
     noEmail: noEmail
   };
 
-  const secondMessageData = {
```

Comment:
secondMessage is no longer necessary because the rejection notification gives you a link to the rejected content, and delete notifications now show the content in the first PM